.travis/run_tests.sh

```diff
@@ -21,7 +21,7 @@ if [ ! -z ${changed_files} ]; then
 		if [ ! -f ${f} ]; then
 			continue
 		fi
-		ts_name=$(grep -oP 'class[\s]+(\K\w+)(?=\(.*\):)' ${f})
+		ts_name=$(grep -oP 'class[\s]+(\KTest\w+)(?=\(.*\):)' ${f})
```

Comment:
@kjakkali Sorry I missed your comment. Do you want me change code? because I am ok with either '[\s]+' or '\s+' as long as its works.